lib/reqTemplate.js

```diff
@@ -4,45 +4,31 @@ var URI = require('swagger-router').URI;
 var TAssembly = require('tassembly');
 var url = require('url');
 var expressionCompiler = require('template-expression-compiler');
+require('core-js/shim');
```

Comment:
This as needed only for a benchmark to work, in production it's not needed as already required way before this code